features/kick_from_team.feature

```diff
@@ -0,0 +1,25 @@
+Feature: allow leaders of a team to kick members from the team
```

Comment:
These scenarios seem very similar to existing remove member scenarios. If the delete feature across different levels of hierarchy is not too big, you could condense into one file.